### PR TITLE
add shellcheck

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
This will only trigger on changed files, so the current violations are not found.

**Comment by evgeni** [comment]
https://github.com/theforeman/forklift/actions/runs/3756166307 has a nice summary showing that this PR fixed 19 issues :)

**Comment by evgeni** [comment]
(no need for this to be WIP, it works by denying new violations, while allowing old ones to persist)

**Review by ekohl — APPROVED** [review]

File: .github/workflows/shellcheck.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```
[the remark this answers is not in the document]
Comment (evgeni) [reply .github/workflows/shellcheck.yml:18]: Not sure, I just copied the example from the action 🙈

**Pull request merged**